doc/source/parameters/cfd/boundary_conditions_cfd.rst

```diff
@@ -75,5 +76,7 @@ where `beta` is a constant and  :math:`(\mathbf{u}\cdot n)_{-}` is :math:`min (0
 
     * ``periodic id`` and ``periodic_direction`` specify the id and direction of the matching periodic boundary condition. For example, if boundary id 0 (located at xmin) is matched with boundary id 1 (located at xmax), we would set ``ìd=0``, ``periodic_id=1`` and ``periodic_direction=0``.
 
-    * ``beta`` is a penalization paramter that is used for both the ``outlet`` and the ``function weak`` boundary conditions. For the outlet boundary conditions, ``beta`` should be close to unity whereas values of ``beta`` of 10 or a 100 can be appropriate for the ``function weak`` boundary condition.
+    * ``beta`` is a penalization parameter used for both the ``outlet``, ``partial slip``, and ``function weak`` boundary conditions. For the outlet boundary conditions ``beta`` should be close to unity, whereas ``beta`` of 10 or a 100 can be appropriate for the ``function weak`` boundary condition. For the ``partial slip`` condition, use high values of ``beta`` (i.e., > 50).
+
+    * ``boundary layer thickness`` (:math:`d_w`) is the parameter applied to the ``partial slip`` boundary condition. It is used to estimate the tangential shear stress :math:`\tau_t = -\mu \frac{u}{d_w}`. For very high ``boundary layer thicknes``, the boundary layer should behave exactly like the ``slip`` condition.
```

Comment:
You should add this parameter to the boundary conditions subsection, with it's default value :)